plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { ContentHeader, InfoCard, Progress } from '@backstage/core-components';
+import {
+  useApi,
+  useRouteRef,
+  useRouteRefParams,
+} from '@backstage/core-plugin-api';
+import { JsonValue } from '@backstage/types';
+
+import { Button, Grid, Typography } from '@material-ui/core';
+import { withTheme } from '@rjsf/core-v5';
+import validator from '@rjsf/validator-ajv8';
+import { JSONSchema7 } from 'json-schema';
```

Comment:
We should avoid this package, it's obsolete for very long time and never properly finished.